lib/events/index.js

```diff
@@ -641,11 +654,11 @@ export default function (self) {
           checkSelectionChange();
           return;
         }
-        if (self.attributes.columnHeaderClickBehavior === 'select') {
+        /*if (self.attributes.columnHeaderClickBehavior === 'select') {
```

Comment:
Can we get rid of this comment entirely? Or do we need it later?

Reply:
ok let me get rid of the comment